src/components/views/rooms/MemberList.js

```diff
@@ -325,27 +348,82 @@ module.exports = React.createClass({
         });
     },
 
-    memberSort: function(userIdA, userIdB) {
-        var userA = this.memberDict[userIdA].user;
-        var userB = this.memberDict[userIdB].user;
+    memberString: function(member) {
+        if (!member) {
+            return "(null)";
+        }
+        else {
+            return "(" + member.name + ", " + member.powerLevel + ", " + member.user.lastActiveAgo + ", " + member.user.currentlyActive + ")";
+        }
+    },
 
-        var presenceMap = {
-            online: 3,
-            unavailable: 2,
-            offline: 1
-        };
+    memberSort: function(now) {
```

Comment:
does this actually need `now`? The relative ordering of users will be the same if you set `now` to a constant (say, zero), and it would save the extra layer of functions.